webui/react/src/pages/TaskList.tsx

```diff
@@ -26,11 +51,28 @@ const TaskList: React.FC = () => {
 
   const hasLoaded = sources.find(src => src.hasLoaded);
 
+  const filteredTasks = filterTasks(loadedTasks, filters, users.data || []);
+
+  const handleFilterChange = useCallback((filters: TaskFilters<CommandType>): void => {
+    storage.set('filters', filters);
+    setFilters(filters);
+  }, [ setFilters, storage ]);
+
   // TODO select and batch operation:
   // https://ant.design/components/table/#components-table-demo-row-selection-and-operation
   return (
     <Page title="Tasks">
-      <TaskTable tasks={hasLoaded ? loadedTasks : undefined} />
+      <div className={css.base}>
+        <div className={css.header}>
```

Comment:
suggestion: there might be some opportunity here to share css between the header here and section options (or dashboard filters wrapper)

Reply:
agreed, let's make it part of the `Page` component to accept left and right options.